styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -90,9 +60,9 @@
   private static final String PARAMETER_DEST = "parameter";
 
   private static final int EXIT_CODE_SUCCESS = 0;
-  private static final int EXIT_CODE_UNKNOWN_ERROR = 1;
-  private static final int EXIT_CODE_ARGUMENT_ERROR = 2;
-  private static final int EXIT_CODE_API_ERROR = 3;
+  private static final int EXIT_CODE_ARGUMENT_ERROR = 1;
```

Comment:
I think 1 for unknown is nicer since whenever i see something other than 1 I assume that there was a specific error.